bin/xud

```diff
@@ -148,6 +148,12 @@ const { argv } = require('yargs')
       type: 'number',
       alias: 'p',
     },
+    'p2p.tor': {
+      describe: 'Enable/disable connections to tor nodes',
+      type: 'boolean',
+      default: false,
```

Comment:
For all booleans, we actually want the default to be `undefined`, the reason being is that if it's not specified via command line we don't want it to override the setting in the config file. If we have a default of false, it be false even if it's true in the config file.